ads/google/imaVideo.js

```diff
@@ -425,6 +425,9 @@ export function imaVideo(global, data) {
     'background-color': 'black',
   });
   videoPlayer.setAttribute('poster', data.poster);
+  if (data.crossorigin != null) {
```

Comment:
`!==`?

Reply:
actually based on CI failure, `data` is not typed and I do want both undefined and null to coalesce, so I can't do this.